config/locales/ca.yml

```
@@ -43,30 +74,46 @@ ca:
       content: "Encara no hi ha cap %{resource_name}."
       link: "Crea'n un/a"
     batch_actions:
-      button_label: "les accions per lots"
-      default_confirmation: "¿Esteu segur que voleu fer-ho?"
-      delete_confirmation: "¿Està segur que desitja eliminar aquests %{plural_model}?"
+      button_label: "Accions per lots"
+      default_confirmation: "Segur que voleu fer-ho?"
+      delete_confirmation: "Segurs que voleu eliminar aquests %{plural_model}?"
       succesfully_destroyed:
-        one: "Va destruir amb èxit 1 %{model}"
-        other: "Va destruir amb èxit %{count} %{plural_model}"
-      selection_toggle_explanation: "(Selecció de Canviar)"
+        one: "1 %{model} eliminat"
+        other: "%{count} %{plural_model} eliminats"
+      selection_toggle_explanation: "(Invertir la selecció)"
       action_label: "%{title} seleccionat"
       labels:
-        destroy: "esborrar"
+        destroy: "Esborrar"
     comments:
-      body: "Cos"
-      author: "autor"
+      created_at: "Creat el"
+      resource_type: "Tipus de recurs"
+      author_type: "Tipus d'author"
+      body: "Missatge"
+      author: "Autor"
       add: "Afegeix comentari"
+      delete: "Elimina comentari"
+      delete_confirmation: "Esteu segurs que voleu eliminar aquest comentari?"
       resource: "Recurs"
       no_comments_yet: "No hi ha comentaris"
-      title_content: "comentaris (%{count})"
+      author_missing: "Anònim"
+      title_content: "Tots els comentaris (%{count})"
       errors:
-        empty_text: "El comentari no es va salvar, el text estava buida."
+        empty_text: "El comentari no s'ha desat, el text estava buit."
     devise:
+      username:
+        title: "Usuari"
+      email:
+        title: "Email"
+      subdomain:
+        title: "Subdomini"
+      password:
+        title: "Contrasenya"
+        password_confirmation: "Confirmació de la contrasenya"
```

Ah I think you meant this. It should also resolve the YAML lint warning.

```suggestion
      password_confirmation: "Confirmació de la contrasenya"
```

---

Yes, you are right, sorry. And I did perform a yamllint locally, but somehow I added lines after and I didn't check again. I'm working on it.